testing/util.go

```diff
@@ -8,8 +8,8 @@ import (
 	"testing"
 )
 
-func CallSite() string {
-	_, file, line, ok := runtime.Caller(2)
+func CallSite(level int) string {
```

Comment:
This is rather brittle. Always was but now even more so.

I reckon we should simply display the entire stack trace instead.